magentov2/plugin/Indodana/PayLater/Gateway/Response/FraudHandler.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Copyright © 2016 Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Indodana\Paylater\Gateway\Response;
+
+use Magento\Payment\Gateway\Data\PaymentDataObjectInterface;
+use Magento\Payment\Gateway\Response\HandlerInterface;
+use Magento\Sales\Model\Order\Payment;
+
+class FraudHandler implements HandlerInterface
```

Comment:
Can you explain why you need this class?

Reply:
This class is not used rigth now. but this class to handle indication fraud entry from the customer.